src/components/Quotes.js

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+
+const Quotes = () => {
+  const [data, setData] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const res = await fetch(
+          'https://api.api-ninjas.com/v1/quotes?category=success',
+          {
+            method: 'GET',
+            headers: {
+              'X-Api-Key': '1MvvMbsdVvSQ3TFuxfyoCA==rjpTXju8ckvX1es5',
+            },
+            contentType: 'application/json',
+          },
+        );
+        const json = await res.json();
+        setData(json);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setData]);
```

Comment:
- [ ] **[OPTIONAL]:** Since `setData` is not going to change between states, I think its safe to remove it from the dependency array! 🤓 🔧 

 ```suggestion
  useEffect(() => {
    const fetchData = async () => {
      setIsLoading(true);
      try {
        const res = await fetch(
          'https://api.api-ninjas.com/v1/quotes?category=success',
          {
            method: 'GET',
            headers: {
              'X-Api-Key': '1MvvMbsdVvSQ3TFuxfyoCA==rjpTXju8ckvX1es5',
            },
            contentType: 'application/json',
          },
        );
        const json = await res.json();
        setData(json);
      } catch (error) {
        setHasError(true);
      }
      setIsLoading(false);
    };
    fetchData();
  }, []);
```

Reply:
Done :wink: 
Thanks a lot for your guide :ok_hand: